psi4/src/psi4/libscf_solver/sad.cc

```diff
@@ -254,7 +254,7 @@ void SADGuess::run_atomic_calculations(SharedMatrix& DAO, SharedMatrix& HuckelC,
 
```

Comment:
This is where it should be 
```suggestion
if(nelec[index]>2*nbf) {
throw PSIEXCEPTION("SAD: Atom " + molecule_->symbol(index) + " has more core electrons than basis functions.");
}
```